keep-ui/app/rules/CorrelationTable.tsx

```diff
@@ -28,6 +28,14 @@ import { useRouter, useSearchParams } from "next/navigation";
 import { FormattedQueryCell } from "./FormattedQueryCell";
 import { DeleteRuleCell } from "./CorrelationSidebar/DeleteRule";
 
+
+const TIMEFRAME_UNITS_FROM_SECONDS= {
```

Comment:
if you are already doing this calculation - why saving the "timeUnit"? isn't it redundant?

Reply:
I'm doing calculations to user-defined units